smart_card_connector_app/src/background.js

```diff
@@ -86,7 +86,8 @@ function launchedListener() {
 }
 
 function openWindow() {
-  GSC.PopupWindow.Server.createWindow(WINDOW_URL, WINDOW_OPTIONS);
+  GSC.PopupWindow.Server.createWindow(WINDOW_URL, WINDOW_OPTIONS,
```

Comment:
nit: Please move the first two arguments into a separate line too: either of the two indentation styles should be used, but not the mix of both (see <https://google.github.io/styleguide/javascriptguide.xml?showone=Code_formatting#Code_formatting>).